paasta_tools/paasta_remote_run.py

```diff
@@ -101,6 +129,22 @@ def main(argv):
     if command:
         overrides_dict['cmd'] = command
 
+    run_id = args.run_id
+    if run_id is None:
+        run_id = ''.join(
+            random.choice(string.ascii_uppercase + string.digits) for _ in range(8))
+        paasta_print("Assigned random run-id: %s" % run_id)
+
+    if args.detach:
```

Comment:
I kinda hate things that background.
Won't this make it literally impossible to debug why it isn't working?
Is this implemented for completeness, do we actually have a use case where we want to hide this (more than cntrl-z?)